ultimatejobweb/templates/sign_in.html

```diff
@@ -19,7 +19,8 @@
  <form method="post">
   {% csrf_token %}
   {{ form.as_p }}
-  <button type="submit">Log In</button>
+  <input type="submit" value="Login">
+<!--   <button type="submit">Log In</button> -->
```

Comment:
remove old code, don't make it a comment

Reply:
Done!

Reply:
Its still here